relay-server/src/endpoints/forward.rs

```diff
@@ -189,29 +177,9 @@ pub fn forward_upstream(
             })
         })
         .and_then(move |result: Result<_, UpstreamRequestError>| {
-            let (is_actix, status, headers, body) =
-                result.map_err(ForwardedUpstreamRequestError::from)?;
+            let (status, headers, body) = result.map_err(ForwardedUpstreamRequestError::from)?;
             let mut forwarded_response = HttpResponse::build(status);
 
-            if is_actix {
-                // For actix-web we called ClientRequestBuilder::disable_decompress(), therefore
-                // the response body is already compressed and the headers contain the correct
-                // content-encoding
-                //
-                // The content negotiation has effectively happened between *our* upstream and
-                // *our* client, with us just forwarding raw bytes.
-                //
-                // Set content-encoding to identity such that actix-web will not to compress again
-                forwarded_response.content_encoding(ContentEncoding::Identity);
-            } else {
-                // For reqwest the option to disable automatic response decompression can only be
-                // set per-client. For non-forwarded upstream requests that is desirable, so we
-                // keep it enabled.
-                //
-                // Essentially this means that content negotiation is done twice, and the response
-                // body is first decompressed by reqwest, then re-compressed by actix-web.
```

Comment:
Can we keep the two lines of this comment somewhere? This is good insight, yet it is not apparent from the code alone.